.github/CONTRIBUTING.md

```diff
@@ -68,8 +82,20 @@ Unkey uses itself to manage its own API keys. To bootstrap the app, run the foll
 
 You need to provide the database credentials as well as the organization ID from clerk as `TENANT_ID`
 
+### Windows
```

Comment:
Let's use tabs here, the default should be Max/linux

https://mintlify.com/docs/content/components/tabs

Reply:
Hmm that completely breaks on Github.

Reply:
If we're doing tabs, can they switch thought the page instead of having to click each one?
Or even better, switch automatically based on the user agent?
If neither's possible, maybe it's better to duplicate the entire process and have one tab at the beginning.